### DISCUSSION TITLE
Plasma printer problem

### DISCUSSION THREAD

**Discussion by 01fiore** [mocaccinoOS/mocaccino#144] [General]
Hello. I download a new ISO Plasma 6 and installed it on a notebook, the system package system-config-printer is missing, i can’t find it in Luet:

moca /home/fiore # luet search kde-base/system-config-printer-kde
 INFO   --- Search results (kde-base/system-config-printer-kde): ---

moca /home/fiore # luet search kde-base/system-config-printer
 INFO   --- Search results (kde-base/system-config-printer): ---

moca /home/fiore # luet search system-config-printer-kde
 INFO   --- Search results (system-config-printer-kde): ---

moca /home/fiore # luet search system-config-printer
 INFO   --- Search results (system-config-printer): ---
![printer](https://github.com/user-attachments/assets/0f7b1ff6-3f3f-45f2-819a-cf585052b764)

$ lspci
00:00.0 Host bridge: Intel Corporation Xeon E3-1200 v3/4th Gen Core Processor DRAM Controller (rev 06)
00:01.0 PCI bridge: Intel Corporation Xeon E3-1200 v3/4th Gen Core Processor PCI Express x16 Controller (rev 06)
00:02.0 VGA compatible controller: Intel Corporation 4th Gen Core Processor Integrated Graphics Controller (rev 06)
00:03.0 Audio device: Intel Corporation Xeon E3-1200 v3/4th Gen Core Processor HD Audio Controller (rev 06)
00:14.0 USB controller: Intel Corporation 8 Series/C220 Series Chipset Family USB xHCI (rev 04)
00:16.0 Communication controller: Intel Corporation 8 Series/C220 Series Chipset Family MEI Controller #1 (rev 04)
00:1a.0 USB controller: Intel Corporation 8 Series/C220 Series Chipset Family USB EHCI #2 (rev 04)
00:1b.0 Audio device: Intel Corporation 8 Series/C220 Series Chipset High Definition Audio Controller (rev 04)
00:1c.0 PCI bridge: Intel Corporation 8 Series/C220 Series Chipset Family PCI Express Root Port #1 (rev d4)
00:1c.3 PCI bridge: Intel Corporation 8 Series/C220 Series Chipset Family PCI Express Root Port #4 (rev d4)
00:1c.4 PCI bridge: Intel Corporation 8 Series/C220 Series Chipset Family PCI Express Root Port #5 (rev d4)
00:1c.5 PCI bridge: Intel Corporation 8 Series/C220 Series Chipset Family PCI Express Root Port #6 (rev d4)
00:1d.0 USB controller: Intel Corporation 8 Series/C220 Series Chipset Family USB EHCI #1 (rev 04)
00:1f.0 ISA bridge: Intel Corporation HM87 Express LPC Controller (rev 04)
00:1f.2 SATA controller: Intel Corporation 8 Series/C220 Series Chipset Family 6-port SATA Controller 1 [AHCI mode] (rev 04)
00:1f.3 SMBus: Intel Corporation 8 Series/C220 Series Chipset Family SMBus Controller (rev 04)
01:00.0 3D controller: NVIDIA Corporation GK106M [GeForce GTX 765M] (rev a1)
03:00.0 Ethernet controller: Qualcomm Atheros Killer E220x Gigabit Ethernet Controller (rev 13)
04:00.0 Unassigned class [ff00]: Realtek Semiconductor Co., Ltd. RTS5227 PCI Express Card Reader (rev 01)
05:00.0 Network controller: Realtek Semiconductor Co., Ltd. RTL8723AE PCIe Wireless Network Adapter

Merci.

**Comment by 01fiore** [comment]
Hello, Plasma 6 update performed this morning on the Notebook without problems.

Old version                           | New version                           | License
apps/steam-1.0.0.79+15                | apps/steam-1.0.0.79+16                | ValveSteamLicense MIT BSD
kernel/mocaccino-lts-full-6.6.51      | kernel/mocaccino-lts-full-6.6.54      |
system/gcc-base-13.3.1+1              | system/gcc-base-13.3.1+2              |
layers/kde-apps-minimal-24.05.2-r1+12 | layers/kde-apps-minimal-24.05.2-r1+13 |
layers/qt-5.15.14+31                  | layers/qt-5.15.14+32                  |
layers/system-x-0.20240923            | layers/system-x-0.20241005            |
kernel/mocaccino-lts-initramfs-6.6.51 | kernel/mocaccino-lts-initramfs-6.6.54 |
layers/plasma-6.1.5                   | layers/plasma-6.1.5+1                 | GPL-2
layers/kde-common-6+19                | layers/kde-common-6+22                |
apps/vim-9.0.2167+35                  | apps/vim-9.1.0366                     | vim
layers/X-1.3+38                       | layers/X-1.3+39                       |
layers/sys-fs-1.1+46                  | layers/sys-fs-1.1+47                  |
kernel/mocaccino-lts-modules-6.6.51   | kernel/mocaccino-lts-modules-6.6.54   |
apps/vlc-3.0.20-r11+15                | apps/vlc-3.0.21                       | LGPL-2.1 GPL-2
apps/thunderbird-115.15.0+2           | apps/thunderbird-128.3.0              | MPL-2.0 GPL-2 LGPL-2.1
apps/firefox-115.15.0+2               | apps/firefox-128.3.0                  | MPL-2.0 GPL-2 LGPL-2.1
apps/flatpak-1.14.8+11                | apps/flatpak-1.14.8+12                | LGPL-2.1+

If possible, I would like to get a message when an important update for KDE is available in the repository, thank you.
![Mocaccino](https://github.com/user-attachments/assets/d0de4a81-43cc-4a77-be95-0e6339eb7c50)

**Reply by joostruis** [reply]
It seems it was already plasma 6.

**Comment by joostruis** [comment]
Can you please open different topics for different questions?

We are not able to (or going to) send messages about updates. We just follow Gentoo stable tree.
I do not understand the printer question.

**Comment by 01fiore** [comment]
For the printer, I can’t get the applet to show up for configuration, I think it’s a bug in KDE.

Where can I see the progress of Plasma via Gentoo, merci.

**Comment by joostruis** [comment]
When I click on "Add printer..." I get this dialog:

![image](https://github.com/user-attachments/assets/aa87aa13-b675-456e-816b-271ab01862fc)

**Comment by joostruis** [comment]
Closing and removing this discussion it is super confusing. If you need help with something please post a new topic and stay on topic.
